platform-sdk/swirlds-unit-tests/common/swirlds-common-test/src/test/java/com/swirlds/common/test/AddressBookTests.java

```diff
@@ -454,6 +454,8 @@ void setNextNodeIdTest() {
     void roundTripSerializeAndDeserializeCompatibleWithConfigTxt() throws ParseException {
         final RandomAddressBookGenerator generator = new RandomAddressBookGenerator(getRandomPrintSeed());
         final AddressBook addressBook = generator.build();
+        // make one of the memo fields an empty string
+        addressBook.add(addressBook.iterator().next().copySetMemo(""));
```

Comment:
I don't see a test that verifies if the address has a memo that it is written in `toConfigText()`. I would like to see a test written reproducing the failure case, then ensuring it passes with the new code.

Reply:
The random address book generator produces random memos, I will make the next address have a memo so the test is insensitive to the random address book generator.

Reply:
What happens if the memo has a comma in it?

Reply:
@rbair23 it should throw a parse exception because there would be 10 items in the line

Reply:
@rbair23 I would expect a parse exception because the number of items in the line would exceed 9

Reply:
Is that the intended behavior? Or should comma's be escaped when being written to the file?

Reply:
I don't know. Do we want to support memos with commas? I'd argue it is not important for this release. We can discuss it as a future enhancement.

Reply:
I mimic what is currently implemented in the parse for the config.txt file.  It splits the content based on newline and then splits each line by a `,`: https://github.com/hashgraph/hedera-services/blob/a6c9bfa34522c8943c357a40fd6cf5a08e2b7efe/platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/config/legacy/LegacyConfigPropertiesLoader.java#L81

Reply:
A better solution would be to have canonical parsing and layout utilities.  But I was informed that the LegacyConfigProperties was deprecated and going the way of the dodo, so I didn't refactor and reuse the code it uses.